darkcluster/src/main/java/com/linkedin/darkcluster/impl/DarkClusterStrategyFactoryImpl.java

```diff
@@ -122,34 +139,63 @@ private DarkClusterStrategy createStrategy(String darkClusterName, DarkClusterCo
       DarkClusterStrategyNameArray strategyList = darkClusterConfig.getDarkClusterStrategyPrioritizedList();
       for (com.linkedin.d2.DarkClusterStrategyName darkClusterStrategyName : strategyList)
       {
-        switch(darkClusterStrategyName)
```

Comment:
Let's make sure to restore the switch() statements for public release

Reply:
+1